examples/quickstart/file_serialization.cpp

```diff
@@ -10,6 +10,7 @@
 #include <hpx/hpx.hpp>
 #include <hpx/hpx_main.hpp>
 #include <hpx/serialization/traits/serialization_access_data.hpp>
```

Comment:
I think as we include the global generated header we can remove the `hpx/serialization/traits/serialization_access_data.hpp` one